src/PanelTraits/Create.php

```diff
@@ -22,12 +25,12 @@ public function create($data)
         $data = $this->decodeJsonCastedAttributes($data, 'create');
         $data = $this->compactFakeFields($data, 'create');
 
-        // ommit the n-n relationships when updating the eloquent item
+        // omit the n-n relationships when updating the eloquent item
         $nn_relationships = array_pluck($this->getRelationFieldsWithPivot('create'), 'name');
         $item = $this->model->create(array_except($data, $nn_relationships));
 
         // if there are any relationships available, also sync those
-        $this->syncPivot($item, $data);
+        $this->createRelations($item, $data);
```

Comment:
Hmm... I think what @tumf87 was planning was to deprecate ```syncPivot()``` entirely in a few point versions, since ```createRelations()``` does the same, but better. Right @tumf87 ?

If so, ```create()``` and ```update()``` are two methods A LOT of people overwrite, and copy-paste the code from vendor to their projects. So no point in providing them with code that will be deprecated.

@tumf87 am I right on this one?